src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataResource.java

```diff
@@ -96,6 +96,16 @@ ResponseEntity updateCasePrimaryCorrespondent(@PathVariable UUID caseUUID, @Path
         return ResponseEntity.ok().build();
     }
 
+    @PutMapping(value = "/case/{caseUUID}/stage/{stageUUID}/updatePrimaryCorrespondent")
+    ResponseEntity updatePrimaryCorrespondent(
```

Comment:
I'm not sure this is needed. Coudn't we use the method above ? 'updateCasePrimaryCorrespondent'

Reply:
Checked with Parth, i thought the mapping would work fine between UUID and String, turns out it does not do so out of the box so I'm happy to accept this solution